testutils/src/main/java/com/microsoft/identity/internal/testutils/nativeauth/NativeAuthCredentialHelper.kt

```diff
@@ -83,4 +83,14 @@ object NativeAuthCredentialHelper {
                 scope
             }
         }
+
+    val nativeAuthInvalidScope: String
+        get() {
+            val scope = BuildConfig.NATIVE_AUTH_INVALID_SCOPE
```

Comment:
Not sure i understand what this build config is needed for

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2125/files#diff-ab55a2e56d7c3096894d487eb0f20b15ab2b2bac7a428a516ae8f778139d6040R56